### DISCUSSION TITLE
Web site accordion

### DISCUSSION THREAD

**Discussion by Emiliano-Bucci** [pmndrs/react-spring#1616] [Support] [answered]
@joshuaellis Hi! I know that this isn't strictly related to react-spring just wanted to know if the accordion component you're using in the site is open sourced, thanks! :)

**Comment by joshuaellis** [comment, marked as answer]
It sure is – https://github.com/pmndrs/react-spring/blob/docs/migrate/docs/src/components/Navigation/index.tsx

**Reply by Emiliano-Bucci** [reply]
@joshuaellis Thanks a lot!

**Reply by Emiliano-Bucci** [reply]
@joshuaellis Very interesting! Just one question, but i guess is related to the ResizeObserver; is there a way to avoid the bouncing/delay of the most external container when the childrens are opened/closed?

**Reply by joshuaellis** [reply]
I guess this example does it better? https://codesandbox.io/s/github/pmndrs/react-spring/tree/master/demo/src/sandboxes/tree

**Reply by Emiliano-Bucci** [reply]
Thanks again, i'll take a look :)
